### PR TITLE
[Docs] Add FAQ and improve navigation

### PR DESCRIPTION
**Pull request by michaelstingl**
## Description
- FAQ page added
- Remove collapsible menu item
- TOC added to index page to improve usage in mobile browser

## How Has This Been Tested?

```
git clone https://github.com/owncloud/ios-app.git
cd ios-app
git checkout docs/improvements+faq
cd docs
yarn install
yarn antora
yarn serve
```

Test http://localhost:8080/ios-app/

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by voroyam — APPROVED** [review]

File: docs/modules/ROOT/pages/ios_faq.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+:toc: right
+
+Here you can find some of the most frequently asked questions about the ownCloud iOS app.
+
+== Usage
+
+=== Compatibility with ownCloud server older than version 10.0
+
+Previous ownCloud iOS app with support for ownCloud server < 10.0 is still available in App Store:  
+https://apps.apple.com/app/id543672169
```
Comment (mmattel) [inline docs/modules/ROOT/pages/ios_faq.adoc:13]: pls use a macro (like the :oauth2-app-url:) for the link

File: docs/modules/ROOT/pages/ios_faq.adoc [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Previous ownCloud iOS app with support for iOS < 12.0 is still available in App Store:  
+https://apps.apple.com/app/id543672169
+
+=== Login with TOTP and other 2FA
+
+ownCloud server must have {oauth2-app-url}[the OAuth2 app] installed, configured, and enabled to use Two-Factor Authentication.
+Please contact your ownCloud administrator for more details.
+
+== Feature Requests
```
Comment (mmattel) [inline docs/modules/ROOT/pages/ios_faq.adoc:25]: An empty headline is sensless and looks bad. Either remove or fill with content.

File: docs/modules/ROOT/pages/ios_faq.adoc [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+== Access full folder via FileProvider, not only single files
+
+This seems like a bigger challenge for us. We track the status here:
+https://github.com/owncloud/ios-app/issues/604
```
Comment (mmattel) [inline docs/modules/ROOT/pages/ios_faq.adoc:30]: pls use a macro (like the :oauth2-app-url:) for the link

File: docs/modules/ROOT/pages/ios_faq.adoc [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+=== Text editing inside the iOS app
+
+Currently this isn't a planned feature. Focus is more to build the best integration in serious 3rd party text editing app.
+
+Here you can find previous discussion:
```
Comment (mmattel) [inline docs/modules/ROOT/pages/ios_faq.adoc:36]: `discussion` --> `discussions`

File: docs/modules/ROOT/pages/ios_faq.adoc [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+https://github.com/owncloud/ios-app/issues/317
```
Comment (mmattel) [inline docs/modules/ROOT/pages/ios_faq.adoc:37]: pls use a macro (like the :oauth2-app-url:) for the link

File: docs/modules/ROOT/pages/ios_faq.adoc [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+=== Translation missing or bug
```
Comment (mmattel) [inline docs/modules/ROOT/pages/ios_faq.adoc:39]: Not clear, needs to be precise
`Translation missing or bug` --> `Missing translations or translation bugs`

File: docs/modules/ROOT/pages/ios_faq.adoc [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+You can help translate in your language or fix a bug.
+
+We use Transifex for translations. Please register for an account and join the global community:
+https://www.transifex.com/owncloud-org/owncloud/mobile-ios-app/
```
Comment (mmattel) [inline docs/modules/ROOT/pages/ios_faq.adoc:44]: pls use a macro (like the :oauth2-app-url:) for the link

File: docs/modules/ROOT/pages/ios_faq.adoc [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+=== Testing new features
+
+Join the beta program on iOS:
+https://owncloud.com/beta-testing/#ios
```
Comment (mmattel) [inline docs/modules/ROOT/pages/ios_faq.adoc:49]: pls use a macro (like the :oauth2-app-url:) for the link

File: docs/modules/ROOT/pages/ios_faq.adoc [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+=== Compatibility with ownCloud server older than version 10.0
+
+Previous ownCloud iOS app with support for ownCloud server < 10.0 is still available in App Store:  
+https://apps.apple.com/app/id543672169
+
+=== Compatibility with iOS versions older than iOS 12.0
+
+Previous ownCloud iOS app with support for iOS < 12.0 is still available in App Store:  
+https://apps.apple.com/app/id543672169
```
Comment (mmattel) [inline docs/modules/ROOT/pages/ios_faq.adoc:18]: pls use a macro (like the :oauth2-app-url:) for the link

**Comment by mmattel** [comment]
@michaelstingl @voroyam please see my comments above. You should file a post fix to address them...

PS: you can delete the branch as it is merged to do a cleanup. Else you have an orphand branch...

**Comment by voroyam** [comment]
Most of what you commented on were improvements.

I am not the author of this PR but I would suggest open a second PR to implement your suggestions @mmattel

**Comment by mmattel** [comment]
> I am not the author of this PR

but you approved it...

**Comment by voroyam** [comment]
Yes, that's called a reviewer :) I have reviewed the document and found no breaking changes.

Michael is the product owner for the iOS docs, so if he thinks that things should be documented a certain way - it's his decision.

As I mentioned in my comment above - if you have some suggestions how further improve this document you can open an issue or make a PR

File: docs/modules/ROOT/pages/ios_faq.adoc (hunk 1 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_faq.adoc:13]: fixed with https://github.com/owncloud/ios-app/commit/4b31f225c301e9a952f3bd9dbed66d8878186328

File: docs/modules/ROOT/pages/ios_faq.adoc (hunk 4 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_faq.adoc:36]: fixed with https://github.com/owncloud/ios-app/commit/4b31f225c301e9a952f3bd9dbed66d8878186328

File: docs/modules/ROOT/pages/ios_faq.adoc (hunk 6 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_faq.adoc:39]: fixed with https://github.com/owncloud/ios-app/commit/4b31f225c301e9a952f3bd9dbed66d8878186328

File: docs/modules/ROOT/pages/ios_faq.adoc (hunk 8 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_faq.adoc:49]: fixed with https://github.com/owncloud/ios-app/commit/4b31f225c301e9a952f3bd9dbed66d8878186328

File: docs/modules/ROOT/pages/ios_faq.adoc (hunk 9 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_faq.adoc:18]: fixed with https://github.com/owncloud/ios-app/commit/4b31f225c301e9a952f3bd9dbed66d8878186328

**Pull request merged**